app/models/user.rb

```diff
@@ -3,6 +3,9 @@ class User < ApplicationRecord
   has_many :comments
   has_many :posts, foreign_key: 'author_id'
 
+  validates :name, presence: true
+  validates :posts_counter, numericality: { only_integer: true }
```

Comment:
- [ ] Nice work here including only integers, however the requirement state that the integer should be greater than or equal to zero, kindly make sure to implement this so negative numbers are excluded from this validation

Reply:
Ok! Haha... I forgot negative integers existed.